docker/server/index.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express')
+
+const defaults = {
+  port: 8080,
+  host: 'http://localhost:3000/graphql',
+  title: 'GraphQL Playground'
+}
+
+const port = +process.env.GRAPHQLPLAYGROUND_PORT || defaults.port
+const host = process.env.GRAPHQLPLAYGROUND_HOST || defaults.host
+const title = process.env.GRAPHQLPLAYGROUND_TITLE || defaults.title
+
+const app = express()
+
+const html = `
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset=utf-8/>
+  <meta name="viewport" content="user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, minimal-ui">
+  <title>${title}</title>
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/css/index.css" />
+  <link rel="shortcut icon" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/favicon.png" />
+  <script src="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/js/middleware.js"></script>
+</head>
+
+<body>
+  <div id="root">
+    <style>
+      body {
+        background-color: rgb(23, 42, 58);
+        font-family: Open Sans, sans-serif;
+        height: 90vh;
+      }
+
+      #root {
+        height: 100%;
+        width: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      .loading {
+        font-size: 32px;
+        font-weight: 200;
+        color: rgba(255, 255, 255, .6);
+        margin-left: 20px;
+      }
+
+      img {
+        width: 78px;
+        height: 78px;
+      }
+
+      .title {
+        font-weight: 400;
+      }
+    </style>
+    <img src='//cdn.jsdelivr.net/npm/graphql-playground-react/build/logo.png' alt=''>
+    <div class="loading"> Loading
+      <span class="title">GraphQL Playground</span>
+    </div>
+  </div>
+  <script>window.addEventListener('load', function (event) {
+      GraphQLPlayground.init(document.getElementById('root'), {
+        endpoint: "${host}"
+      })
+    })</script>
+</body>
+
+</html>
+`
+
+app.get('/', (req, res) => res.send(html))
+
+app.listen(port, function(error) {
+  if (error) console.error(error)
+  console.log(
+    `GraphQL Host set to ${host} ${
+      host === defaults.host ? '(Default)' : ''
+    }`,
+    `\n\nServing the GraphQL Playground on http://localhost:${port}/playground ${
```

Comment:
does it only listen on 127.0.0.1 ? if not be default there should be an option to make it listen on 0.0.0.0 or something else (so that it works with a reverse proxy)